infrastructure/builder/src/generate/generators/docs-tocs.js

```diff
@@ -70,8 +70,11 @@ async function readGeneratedDocs() {
     // Rename keys to include the section, tier and service name
     // e.g., docs/intro -> reference/integrations/github/docs/intro
     Object.keys(mdFiles).forEach(oldKey => {
-      const path = `reference/${metadata.tier}/${projectName}/${oldKey}`;
+      let path = `reference/${metadata.tier}/${projectName}/${oldKey}`;
 
+      if(oldKey === "references/events" || oldKey === "references/api"){
+        path = `reference/${metadata.tier}/${projectName}/v1/${oldKey}`;
+      }
```

Comment:
I think there's an earlier phase where these `mdFiles` are created - that would be the place to do this naming.  There, you'll have access to the reference document itself, which has an `apiVersion` property that should be used to give the version -- in theory, someday that may be something other than `v1` :)